### ISSUE TITLE
How long should i wait for initial setup for kubuntu20.04.2

### ISSUE THREAD

**Issue by shintian-kubuntu** [hakuna-m/wubiuefi#275]
I use wubi200420r342.exe

I was able to install ubuntu 20.04.2.0 with wubiuefi.
But, I can't install kubuntu 20.04.2.0 with wubiuefi.

 I was able to select kubuntu and reboot on  wubi200420r342.exe.
After reboot, I can go as far as the kubuntu wallpaper is displayed.
Even if I wait for a few minutes, only the wallpaper is displayed.

How can I start the initial setup?

**Comment by hakuna-m** [comment]
> Even if I wait for a few minutes, only the wallpaper is displayed.
> 
> How can I start initial setup?

see [release notes](https://github.com/hakuna-m/wubiuefi/releases/tag/200420r342):

> * (*) Kubuntu has no progress bar during second part of installation.

So second part of Kubuntu installation only provides a wallpaper without progress bar.

**Comment by shintian-kubuntu** [comment]
Thank you for your reply.
I saw what was written in the release notes.
That means that wubiuefi can't initialize kubuntu.
wubiuefi can't finished install kubuntu.
Is it correct?

or,,,if i adding the --queit option , I can install kubuntu???

**Comment by shintian-kubuntu** [comment]
Maybe if I select Verbose mode in the Grub2menu, will kubuntu install be completed?

**Comment by hakuna-m** [comment]
> That means that wubiuefi can't initialize kubuntu.
> wubiuefi can't finished install kubuntu.
> Is it correct?

No, it isn't correct. The Kubuntu installer for automatic installations has no slide show and graphical progress bar. But installation should run as a background process.

Background process needs as long as Ubuntu installer with progress bar.

If installation does not work, Kubuntu installer should report warnings or error messages. e.g. see #209. But #209 has been solved.

**Comment by shintian-kubuntu** [comment]
Thank you for your reply.
I was misunderstanding. Thank you for making me aware.

Next time, I try kubuntu install again.
During second part of installation of kubuntu, I will wait a long time. About 30 minutes.
Thank you.
